eradiate/thermoprops/util.py

```diff
@@ -252,60 +489,63 @@ def _to_regular(mesh, atol):
         with the smallest number of layers. Depending on the value of ``atol``,
         the resulting mesh size can be large.
 
-    Parameter ``mesh`` (array):
+    Parameter ``mesh`` (:class:`numpy.ndarray`):
         Irregular altitude mesh with values sorted in increasing order.
 
-    Parameter ``atol`` (float):
+    Parameter ``atol`` (:class:`float`):
         Absolute tolerance used in the conversion.
 
-    Returns -> array:
+    Returns -> :class:`numpy.ndarray`:
         Regular altitude mesh.
     """
 
     n, _ = _find_regular_params_gcd(mesh, atol)
     return np.linspace(start=mesh[0], stop=mesh[-1], num=n)
 
 
-def _find_regular_params_gcd(mesh, unit_number=1.):
-    r"""Finds the parameters (number of cells, constant cell width) of the
+def _find_regular_params_gcd(mesh, unit_number=1.0):
+    """Finds the parameters (number of cells, constant cell width) of the
     regular mesh that approximates the irregular input mesh.
 
-    The algorithm finds the greatest common divisor (GCD) of all cells widths in
-    the integer representation specified by the parameter ``unit_number``.
+    The algorithm finds the greatest common divisor (GCD) of all cells widths
+    in the integer representation specified by the parameter ``unit_number``.
     This GCD is used to define the constant cells width of the approximating
     regular mesh.
 
     .. warning::
         There are no safeguards regarding how large the number of cells in the
         regular mesh can be. Use the parameter ``unit_number`` with caution.
 
-    Parameter ``mesh`` (array):
+    Parameter ``mesh`` (:class:`ndarray`):
         1-D array with floating point values.
         Values must be sorted by increasing order.
 
-    Parameter ``unit_number`` (float):
+    Parameter ``unit_number`` (:class:`float`):
         Defines the unit used to convert the floating point numbers to integer.
         numbers.
 
         Default: 1.
 
-    Returns -> int, float:
-        Number of points in the regular mesh and the value of the constant cells
-        width.
+    Returns -> :class:`tuple`:
```

Comment:
Please revert to get closer to the tuple typing spec rationale [https://docs.python.org/3/library/typing.html#typing.Tuple].